src/router/Routes.tsx

```diff
@@ -28,6 +31,7 @@ const Router = () => {
                     <AuthRoute path={"/userhome"} component={Logged}/> //Requires being logged in
                     <Route path={"/signup"} component={signUp}/>
                     <AdminRoute path={"/adminhome"} component={AdminHome}/> //Requires admin role
+                    <Route path={'/editprofile/:userId'} component={EditProfile}/>
```

Comment:
This path could be only `/profile`? Because the view and the edit profile are the same screens.

Reply:
done